### ISSUE TITLE
Support for additional properties

### ISSUE THREAD

**Issue by Flaxoos** [Flaxoos/extra-ktor-plugins#29]
The plugin should also allow configuration of additional properties, such as SASL, SSL, and Security.

see https://github.com/Flaxoos/extra-ktor-plugins/issues/26

**Comment by diegopatinor** [comment]
Hi @Flaxoos, is this not developed yet?

**Comment by Flaxoos** [comment]
Hi @diegopatinor unfortunately not yet, im busy with other tasks on another project of mine. Would you like to take this on? It's probably pretty straightforward and just a matter of extending the configuration classes

**Comment by Flaxoos** [comment]
@diegopatinor this has been addressed in v2.1.0 https://github.com/Flaxoos/extra-ktor-plugins/releases/tag/v2.1.0, you can now do for example:

```kotlin

 install(Kafka) {
                   ...
                    consumer {
                            additional {
                                ANY_PROPERTY_STRING("some value)
                            }
                    }
                    commonSsl {
                        broker {
                            protocol = "TLSv1.2"
                            endpointIdentificationAlgorithm = ""
                            trustStore {
                                location = "path_to_file"
                                password ="password"
                            }
                            keyStore {
                                location = "path_to_file"
                                password ="password"
                            }
                            keyPassword = PASSWORD
                        }
                        schemaRegistry {
                            protocol = "TLSv1.2"
                            endpointIdentificationAlgorithm = ""
                            trustStore {
                                location = "path_to_file"
                                password ="password"
                            }
                            keyStore {
                                location = "path_to_file"
                                password ="password"
                            }
                            keyPassword = PASSWORD
                        }
                    }
              }
```

**Comment by Flaxoos** [comment]
Closing as fixed
